learn/getting-started/writing-a-restful-api-with-ballerina.md

```diff
@@ -24,11 +24,13 @@ To complete this tutorial, you need:
 
 ## Designing the Two Endpoints 
 
-The first endpoint is about getting data from the service as well as adding data to the service. Therefore, the service should handle both HTTP GET and POST requests. The GET request is to get data whereas the POST request is to add data. In the case of the GET request, the response should be 200 OK whereas in the case of POST request the response should be 201 created.
+The first endpoint is about getting data from the service as well as adding data to the service. Therefore, the service should handle both HTTP GET and POST requests.
+- The GET request is to get data, and the response should be `200 OK`.
```

Comment:
```suggestion
- The GET request is to get data and the response should be `200 OK`.
```

Reply:
What is the reasoning behind this? From Grammarly:

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/16300038/151652435-91502c91-905c-4a03-ba93-3f9020c7bf06.png">

Reply:
+1 as this is a compound sentence.